src/VideoPlayer/index.js

```diff
@@ -84,6 +95,14 @@ const fireOnConsumer = (event, args) => {
     consumer.fire('$videoPlayer' + event, args, videoEl.currentTime)
     consumer.fire('$videoPlayerEvent', event, args, videoEl.currentTime)
   }
+  if (event === events['timeupdate'] && subtitles.enabled) {
```

Comment:
Currently, there is no API to cover this use case:
- The user enables subtitles (and the app calls `VideoPlayer.openSubtitles()`)
- The user disables subtitles again -> `clearSubtitles` can be called but it will still trigger this parsing

Reply:
`subtitles.enabled` is set to `false` on `clearSubtitles`, so the parser won't be called after clear.

Reply:
Ah, check